build-hugo.sh

```
@@ -0,0 +1,31 @@
+
+# Install Hugo Extended
+CGO_ENABLED=1 go install -tags extended github.com/gohugoio/hugo@$HUGO_VERSION
+
+echo 'export PATH=$PATH:$GOBIN' >> ~/.bashrc
+echo 'export PATH=$PATH:$GOBIN' >> ~/.zshrc
+
+echo 'export PATH=$PATH:$HOME/go/bin' >> ~/.bashrc
+echo 'export PATH=$PATH:$HOME/go/bin' >> ~/.zshrc
+
+# Source the bashrc or zshrc file (depending on the shell)
+source ~/.bashrc
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)